src/fetch/transfer_file.py

```diff
@@ -403,11 +403,15 @@ def get_cow_rewards(dune: DuneAPI, period: AccountingPeriod) -> list[Transfer]:
     solver_num_trades: dict[str, int] = {
         row["solver"].lower(): int(row["num_trades"]) for row in dune_trade_counts
     }
-    db_solvers = set(cow_rewards_df.receiver)
-    dune_solvers = set(solver_num_trades.keys())
-    assert (
-        db_solvers == dune_solvers
-    ), f"solver sets disagree: {db_solvers.symmetric_difference(dune_solvers)}"
+    duplicates = pd.concat(
+        [
+            pd.DataFrame(dune_trade_counts),
+            cow_rewards_df[["receiver", "num_trades"]].rename(
+                columns={"receiver": "solver"}
+            ),
+        ]
+    ).drop_duplicates(keep=False)
+    assert len(duplicates) == 0, f"solver sets disagree: {duplicates}"
     for _, row in cow_rewards_df.iterrows():
         solver = row.receiver.lower()
         assert row.num_trades == solver_num_trades[solver], "invalid trade count!"
```

Comment:
I think the proposed overlap calculation should replace this part as well. What's your take on this?

Reply:
Not sure what you mean... This asserts trade counts. I am not super familiar with pandas if you'd like to show me what you were thinking.

Reply:
`pandas.DataFrame.drop_duplicates` looks at both the solvers and trade counts and only drops it if both match. E.g.: It only drops two rows if they have the same solver address and same amount of trade counts. 

I assume this is what is being done there?

Reply:
No this is a comparison with the trade count from dune and the order book

Reply:
> No this is a comparison with the trade count from dune and the order book

That's what the proposed code does as well.

Reply:
Ok I'll check it out later.

Reply:
Oh. So you're saying we don't need this anymore.

Reply:
The code above asserts

> Oh. So you're saying we don't need this anymore!
> 
> Nice!

That's right! Thanks!

Reply:
Pandas is our friend!

Reply:
GeFixed! Thanks for your suggestion here.